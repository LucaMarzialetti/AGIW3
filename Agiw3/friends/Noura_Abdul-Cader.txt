--url_fb https://www.facebook.com/noura.abdulcader
--friends
VM35047:3 Nabeela H Yusuf
VM35047:3 Safiyas Cakes
VM35047:3 Rosemary Read
VM35047:3 Mona Abdul-Cader
VM35047:3 Krupa Jethwa
VM35047:3 Hannah Rose
VM35047:3 Naadiya Salihue
VM35047:3 Sara Alif
VM35047:3 Mariam Abdul-Cader
VM35047:3 Anjum Kasmani
VM35047:3 Sara Hyder-Sheikh
VM35047:3 Colin Read
VM35047:3 Ahmad Huzair
VM35047:3 Aatif Nawaz
VM35047:3 Sophie Walker
VM35047:3 Sumayya Abdul-Cader
VM35047:3 Sharma Charles
VM35047:3 Andy Tingle
VM35047:3 Becca Degnarain
VM35047:3 Maimuna Bappa
VM35047:3 MuhammadAli Macci
VM35047:3 Ramo Alif
VM35047:3 Basheera Samad
VM35047:3 Nada Omar
VM35047:3 Kavish Hussain Sangji
VM35047:3 Ilma Gaffoor
VM35047:3 Jai Patel
VM35047:3 Samira Umm Maryam
VM35047:3 Dru Goradia
VM35047:3 Jemina Punni
VM35047:3 Stella Stylianidou
VM35047:3 Paul Robinson
VM35047:3 Suraya OZ
VM35047:3 Elita Karim
VM35047:3 Hajra Ali
VM35047:3 Amina Rajput
VM35047:3 Shahlaa Yasmin
VM35047:3 Harald Köpping Athanasopoulos
VM35047:3 Jessy Papasavva
VM35047:3 Raoul Colvile
VM35047:3 Mohamed Joseph El-Ali
VM35047:3 Beverley Basketcase James
VM35047:3 Aniqah Junaideen
VM35047:3 Farah Mohiuddin
VM35047:3 Charlotte Brunsdon
VM35047:3 Sara Amanulla
VM35047:3 Suna Mirza
VM35047:3 Naomi Lecomte
VM35047:3 Jasmine Chan
VM35047:3 Adeeba Tazeen
VM35047:3 Nikita Kotecha
VM35047:3 Alzira Peres
VM35047:3 Callum Hassall
VM35047:3 Emma Kennedy
VM35047:3 Nayan Gami
VM35047:3 Kinna Patel
VM35047:3 Jennifer Ashley
VM35047:3 Terry Murphy
VM35047:3 Divya Rajamohan
VM35047:3 Aman Bhatti
VM35047:3 Tasnim Nazeer
VM35047:3 Mubasshira Rakhange
VM35047:3 Azara Shareefa Zahir
VM35047:3 Alice Hussey
VM35047:3 Ilham Aboo
VM35047:3 Tazkera Umm Zaynab
VM35047:3 Hadeer S
VM35047:3 Sara Haque
VM35047:3 David Broome
VM35047:3 Midya Shakir
VM35047:3 Anisa Tyebally
VM35047:3 Narjis Khan
VM35047:3 Jenny John
VM35047:3 Amjad Khan
VM35047:3 Arnita Manandhar
VM35047:3 Ayesha R. Rafiq
VM35047:3 Imran Khan
VM35047:3 Jules Carson
VM35047:3 Amir Tyebally
VM35047:3 Sajid Faiz
VM35047:3 Tom MacL
VM35047:3 Aisha Corning
VM35047:3 Naila Ahmad
VM35047:3 Sulaf Hussein
VM35047:3 Hafsah Adham
VM35047:3 Fiona Noble
VM35047:3 Nuha Ghouse
VM35047:3 Luise Birgelen
VM35047:3 Nia Barnabie
VM35047:3 Saima Jaffar
VM35047:3 Krishan Kanhaiya Gandhi
VM35047:3 Kate Murray
VM35047:3 Stavros Avery
VM35047:3 Seb Orton
VM35047:3 Katharina Kunze
VM35047:3 Shiyana Eliyas
VM35047:3 Shakira Parkar
VM35047:3 Ayesha Abdeen
VM35047:3 Sam Handy
VM35047:3 Hasna Ghany-Yusuf
VM35047:3 Persona Artistry
VM35047:3 Huma Malik
VM35047:3 Asma Fuzail
VM35047:3 Diana مريم Stelzer
VM35047:3 Ellaine Gelman
VM35047:3 Tobias Akash Shanker
VM35047:3 Abdullah Mohamed
VM35047:3 Jennifer Burgess
VM35047:3 Shanaz Yusufkhan
VM35047:3 Ankita Patel
VM35047:3 Stefan Alok Shanker
VM35047:3 Nida Adnan Nanjee
VM35047:3 Karmjit Kaur
VM35047:3 Abbas A
VM35047:3 Savvas Papasavva
VM35047:3 John Page
VM35047:3 Louiza Amira Chekhar
VM35047:3 Kyriacos Papasavva
VM35047:3 Edmund Page
VM35047:3 Lisa Webber
VM35047:3 Leon Farr
VM35047:3 Sophie Louise
VM35047:3 Florian HT Albert
VM35047:3 Sana'a Abbas
VM35047:3 Hiba Mohamad
VM35047:3 Javeria Hassan Haque
VM35047:3 Sheherazadh Ishaq
VM35047:3 Mohamed Siraj
VM35047:3 Nicola Mollat
VM35047:3 Jaffar Al-Rikabi
VM35047:3 George Castle
VM35047:3 Nikki Shalini P
VM35047:3 Azra Khan
VM35047:3 Robert Woodward
VM35047:3 Sanna Waseem
VM35047:3 Hameeda Khan
VM35047:3 Sarah Malik
VM35047:3 Nusrat Ahmad
VM35047:3 Samia Ahmed
VM35047:3 Ruheena Chaudhry
VM35047:3 Carly Becker
VM35047:3 Jasmeet K Panesar Marway
VM35047:3 Katy Boo
VM35047:3 Aneeza Haleem
VM35047:3 Vasita Patel
VM35047:3 Haifa Shameem
VM35047:3 Sneha Patel
VM35047:3 Richie Harman
VM35047:3 Olivia Bailey
VM35047:3 Mohd Hizami Mohd Iskandar
VM35047:3 Jeeves Wij
VM35047:3 Rahma Amanulla
VM35047:3 Falguni Upadhyaya
VM35047:3 Alex Dowty
VM35047:3 Madiha Haque
VM35047:3 Jane Stinson
VM35047:3 Salmah Shahnawaz
VM35047:3 Sheena Patel
VM35047:3 Carol Yenovkian
VM35047:3 Mehrun Feisal
VM35047:3 Anissara Aziz
VM35047:3 Hina Ahmad Gilani
VM35047:3 Faith Dore
VM35047:3 Georgina Sword-Daniels
VM35047:3 Anne Irfan
VM35047:3 Mahmood Haque
VM35047:3 Terri Bull
VM35047:3 Sana Hawai
VM35047:3 Ally Chhatwal
VM35047:3 Sonya Nikolosina
VM35047:3 Craig Eggett
VM35047:3 Jenny Gali
VM35047:3 Anna Lachowska
VM35047:3 Luke Johnson
VM35047:3 Swapna Shankar
VM35047:3 Nadhiya Ito
VM35047:3 Shagufta Sami
VM35047:3 Faiz Kha
VM35047:3 Irene Anderson
VM35047:3 Humaira Khwaja
VM35047:3 Meeta Kalyanji Gohel
VM35047:3 Saadia Amanulla
VM35047:3 Afrah S Sait
VM35047:3 Kanika Singh
VM35047:3 Fathima Amaana Faleel
VM35047:3 Aneesa Ifteqar
VM35047:3 Shirin Yoosoofsah
VM35047:3 Sita Vekaria
VM35047:3 Salwa Parkar
VM35047:3 Shehla Ahmad
VM35047:3 Sachin Shah
VM35047:3 Heena B
VM35047:3 Farah Cader
VM35047:3 Gouljan Arslan
VM35047:3 Attiya Anwar Haque
VM35047:3 Saher Baig
VM35047:3 Claire Bendix
VM35047:3 Yvonne Perkins
VM35047:3 Maryam Malik
VM35047:3 Danielle Short
VM35047:3 Sonia Umm Yousuf
VM35047:3 Dana Zuhair
VM35047:3 Jinendra Ekanayake
VM35047:3 Dolly Shah
VM35047:3 Asilah Motala
VM35047:3 Martha Parkhurst
VM35047:3 Abess El-Ali
VM35047:3 Swathi Ramesh
VM35047:3 Lucille Varghese Samuel
VM35047:3 Komal Patel
VM35047:3 Hafsah Othman
VM35047:3 Ali Aslam
VM35047:3 Kalika Sunger
VM35047:3 Dolapo Osonowo
VM35047:3 Hemali Mehta
VM35047:3 Greg Scannell
VM35047:3 Dhimal Varsani
VM35047:3 Pooja Patel
VM35047:3 Bhavika Vaghela
VM35047:3 Nilesh Vyas
VM35047:3 Lima Davod
VM35047:3 Jharna Bhatia
VM35047:3 Lila Fernando
VM35047:3 Hiten Varsani
VM35047:3 Anita Abram
VM35047:3 Keran Sandhu
VM35047:3 Salma Yusuf
VM35047:3 Nadia Oosthuizen
VM35047:3 Vibuhti Parmar
VM35047:3 David Joseph
VM35047:3 Rachael S Khan
VM35047:3 Disha Raja
VM35047:3 Arshiya Vahid Shaikh
VM35047:3 Alvin Chitroda
VM35047:3 Amira Nassr
VM35047:3 Sandhya Naidoo
VM35047:3 Hannah Ayesha
VM35047:3 Mark Dowds
VM35047:3 Lauris Blockerÿe
VM35047:3 Nirav Parekh
VM35047:3 Rasha Jahfar
VM35047:3 Sheik Zahir
VM35047:3 Naila Malik
VM35047:3 Bushra Zaheer
VM35047:3 Mohammad Shakeel
VM35047:3 Laura HW
VM35047:3 Mohamed Feisal
VM35047:3 Samiya Razzaq
VM35047:3 Prophetic Guidance
VM35047:3 Nabilah Tyebally
VM35047:3 Jacqui Bernuzzi
VM35047:3 Dion Hamilton
VM35047:3 Daksha Jaysavani
VM35047:3 Naomi Elisabeth
VM35047:3 Kel Thompson
VM35047:3 Fowzun Abdul-Cader
VM35047:3 Jo Fisher
VM35047:3 Princess-Grace Davies
VM35047:3 Memake David
VM35047:3 Hudha Abdul Careem
VM35047:3 Paul Blessing
VM35047:3 Iama Sportylady
VM35047:3 Khaled Shaaban
VM35047:3 Olly Richardson
VM35047:3 Safiya Amanulla
VM35047:3 Greg Lawson
VM35047:3 Saffron Finch
VM35047:3 Zaujat Ahmed
VM35047:3 Seema Towfeeq
VM35047:3 Mark Everitt
VM35047:3 Ziaudeen Abdul-cader
VM35047:3 Barbara Lachowska
VM35047:3 Francois Holmey
VM35047:3 Nada Ali
VM35047:3 Mark Edwards
VM35047:3 Huw Morrison
VM35047:3 Sruthi Cv
VM35047:3 Dalena Mathew
VM35047:3 Sajana Krishna
VM35047:3 Tariq Ali
VM35047:3 Zach McCollough
VM35047:3 Zee Shan
VM35047:3 Herson Samayoa
VM35047:3 Muneer Malik
VM35047:3 Aisha Rahat Husain
VM35047:3 Husna Khan
VM35047:3 Shahida Macci
VM35047:3 Kadir Abdul
VM35047:3 Jaimy Jose
VM35047:3 Farah Sabri
VM35047:3 Hind Essoussi
VM35047:3 Sofia Moiz Tyebally
VM35047:3 Sara Moiz Tyebally
VM35047:3 Hishma Faleel
VM35047:3 Sayyed Anj
VM35047:3 Faathima Sufi-ismail
VM35047:3 Mohamed Khizer
VM35047:3 Ulfath Munavvara
VM35047:3 Saif Feisal
VM35047:3 Rachel Morris
VM35047:3 Rimla Akhtar
VM35047:3 Daniel Medawar
VM35047:3 Mohammed Saud Khan
VM35047:3 Sarah Kelly
VM35047:3 Rajiv Khatri
VM35047:3 Alan Wahid
VM35047:3 Elena Giacon
VM35047:3 Natasha Holcroft-Emmess
VM35047:3 Suzana Ali Hasan Thaver
VM35047:3 Triska Hamid
VM35047:3 Zeanne Ayesha
VM35047:3 Zarah Haniff
VM35047:3 Samir Anverdeen
VM35047:3 Husnain Fatima
VM35047:3 Shaun Palmer
VM35047:3 Mathew Du Val
VM35047:3 Lizzie Pennant
VM35047:3 Helen Field
VM35047:3 Vivienne Inmonger
VM35047:3 Fathima Zahrah
VM35047:3 Arya Suresh Babu
VM35047:3 Samira Sayed
VM35047:3 Scott Seamons
VM35047:3 Nina Naghmana Khan
VM35047:3 Ahmed Shehan Feisal
VM35047:3 Shereen Jose
VM35047:3 Alex Smyth-Osbourne
VM35047:3 Laura Williams
VM35047:3 Hayley Boot
VM35047:3 Prithpal Singh Bhachu
VM35047:3 Hanif Zahir
VM35047:3 Sal Man
VM35047:3 Patricia Sullivan
VM35047:3 Navira Shehan
VM35047:3 Henry Fisher
VM35047:3 Narmila Alif
VM35047:3 Phil Robinson
VM35047:3 Sumoyee Basu
VM35047:3 Ruben Leavitt
VM35047:3 Zainab Shahnawaz
VM35047:3 Nazar Khan
VM35047:3 Abdur Nazimudeen
VM35047:3 Imran Iqbal
VM35047:3 Shiri Ri
VM35047:3 Ben Chappell
VM35047:3 Lauren Mayumi Brodsky
VM35047:3 Amrita Zahir
VM35047:3 Reza Rafiq
VM35047:3 Ismail Salihue
VM35047:3 Aleena Mishal Khan
VM35047:3 Afifah T Waseem
VM35047:3 Kate Wright
VM35047:3 Alastair Vettese
VM35047:3 Ciaran O'Sullivan
VM35047:3 Amrit Zahir
VM35047:3 Elwin Wilson
VM35047:3 James Green
VM35047:3 Edison Thomas
VM35047:3 Tasneem Ashraf
VM35047:3 Rifki Mohamed
VM35047:3 NiSha GopalkriShnan
VM35047:3 Azman Khan
VM35047:3 Shiham Khan
VM35047:3 Maariyah Bandukara
VM35047:3 Aneta Morávková
VM35047:3 Susan Lazar
VM35047:3 Maryam Ijaz Shah
VM35047:3 Jolin Joseph
VM35047:3 James Green
VM35047:3 Zoë Rawson
VM35047:3 Jessica Keal
VM35047:3 Hayzil Bee
VM35047:3 Dilshad Sadiq
VM35047:3 Andrew Bershadski
VM35047:3 Rory Milligan
VM35047:3 Jonny Medland
VM35047:3 Saheba Shahed
VM35047:3 Ellie Marie
VM35047:3 Vittoria Ando
VM35047:3 Nadha Ahmed
VM35047:3 Meera Joshi
VM35047:3 Adam Haigala
VM35047:3 Omar Abdel-Mannan
VM35047:3 Hannah Martinez
VM35047:3 Imran Bhatia
VM35047:3 Ruth Campbell
VM35047:3 Shanaka Jayasuriya
VM35047:3 Nathan Koblintz
VM35047:3 James Campbell
VM35047:3 Saher Iqbal
VM35047:3 Dipesh Patel
VM35047:3 Prakash Pindoria
VM35047:3 Amy Khan
VM35047:3 Farah Noor
VM35047:3 Jennifer Coles
VM35047:3 Sasha Willmott
VM35047:3 Nadia Macci
VM35047:3 Elizabeth Staehr
VM35047:3 Vhari Pontin
VM35047:3 Saima Mohammad Qais
VM35047:3 Shabnam Farook
VM35047:3 Avani Acharya
VM35047:3 Maanas Jain
VM35047:3 Safra Ali
VM35047:3 Hoda Layla Nassr
VM35047:3 Haasher Ma'nawi
VM35047:3 Nafisa Sufi-Ismail
VM35047:3 Partha S. Mudgil
VM35047:3 Jessica Bush
VM35047:3 James Lloyd-Thomas
VM35047:3 Alicia Tew
VM35047:3 Sabeeh Feisal
VM35047:3 Alan Thomas
VM35047:3 Anna Fionnuala James Hyde
VM35047:3 Nasooha Junaidean
VM35047:3 Afshan Zuberi Oosman
VM35047:3 Inshi Sadiq
VM35047:3 Maryam Sufi-Ismail
VM35047:3 Peter Morcos
VM35047:3 Jo Hemmings
VM35047:3 Prit Pan
VM35047:3 Emily Putz
VM35047:3 Thomas Hayward
VM35047:3 Khushbu Gandhi
VM35047:3 Mikaela Taberner
VM35047:3 Taqdis Dhinda
VM35047:3 Laura Charlotte Rowan MacPhee
VM35047:3 Hashman Mahnavi
VM35047:3 Dhruti Pandya
VM35047:3 Lorna Greenwood
VM35047:3 Qian Li
VM35047:3 Emily Duggan
VM35047:3 ToxicFly Mem-z
VM35047:3 Zoë Hoster
VM35047:3 Hani Al-Mousawi
VM35047:3 Philipp Gnatzy
VM35047:3 Jason Craig
VM35047:3 Zahara Amanulla
VM35047:3 Matilda Williams
VM35047:3 Matilda Macduff
VM35047:3 Rumaiz Khan
VM35047:3 Philip Hutchinson
VM35047:3 Anna Furness
VM35047:3 Sazan Meran
VM35047:3 Sepideh Bazazi
VM35047:3 Tina BN
VM35047:3 Michael Papasavva
VM35047:3 Jalpa Nandha
VM35047:3 Dipesh Balsara
VM35047:3 Sarah Connell
VM35047:3 Sarah Shafi
VM35047:3 Mufidah Motala
VM35047:3 Sajeevithan Puvaneswaran
VM35047:3 Narmeen Rehman
VM35047:3 Sohan Bhudia
VM35047:3 Richa Pathak
VM35047:3 Pari Mamtora
VM35047:3 Omotola Adunni Haskell
VM35047:3 Rico Lee
VM35047:3 Armaan Khan
VM35047:3 Tish Gunton
VM35047:3 Sophie Pitman
VM35047:3 Marcus Daniel Trotter
VM35047:3 Jamie Hill
VM35047:3 Krupa Vekaria
VM35047:3 Anna Mertens
VM35047:3 Mehul Patel
VM35047:3 Krish Kunam
VM35047:3 Chandni Chadha
VM35047:3 Sachin Patel
VM35047:3 Catherine Garsed
VM35047:3 Sarah De-Salis
VM35047:3 Emma Grisdale
VM35047:3 Kate Rudman
VM35047:3 Lizzie Floyd
VM35047:3 Catherine Carroll
VM35047:3 Taha Qais
VM35047:3 Harry Whitehead
VM35047:3 Andrew Rudhall
VM35047:3 Ditiksha Varsani
VM35047:3 Shazna Dilshad Sadiq
VM35047:3 Carla Walker
VM35047:3 Helena Foster
VM35047:3 Yasir Khan
VM35047:3 Nisha Bose
VM35047:3 Purvesh Patel
VM35047:3 Amanda Julius
VM35047:3 Anthony Gamble
VM35047:3 Phoebe Bown
VM35047:3 Rosemin Abdeen
VM35047:3 Jumana Shariff
VM35047:3 Leya Khan
VM35047:3 Vincent Knowles
VM35047:3 Loris Shala
VM35047:3 Anabel Nirupa
VM35047:3 Kasia Puścian
VM35047:3 Louise WA
VM35047:3 Sana Qais
VM35047:3 Ben Wood
VM35047:3 Munazza Ahsan
VM35047:3 Mimi Lubelle
VM35047:3 Prince Smith
VM35047:3 Kay Charlene
VM35047:3 Nabila Abdul Malik
VM35047:3 Turab Shah
VM35047:3 Mehul Patel
VM35047:3 Charlotte Nolan
VM35047:3 Fahd Mahmood
VM35047:3 Farrah Ahmed
VM35047:3 Sana Qais
VM35047:3 Rini Banerjee
VM35047:3 Vikesh Gami
VM35047:3 Sivananthi T Sujeen
VM35047:3 Alina Fazal
VM35047:3 Safia Hyder
VM35047:3 Shamim Chowdhury
VM35047:3 Nikhil Patel
VM35047:3 Monther Al-Hakim
VM35047:3 Ali Hashim
VM35047:3 Rikki Bhachu
VM35047:3 Dean R.
VM35047:3 Shoaib Zubar
VM35047:3 Anita Tofts
VM35047:3 Azim Wazeer
VM35047:3 Nafeesah Stewart
VM35047:3 Manish Kerai
VM35047:3 Saqib Kazmi
VM35047:3 Arunan Sundarathasan
VM35047:3 Mohammed Moussa
VM35047:3 Yas Mahmoud
VM35047:3 Helen Marouli
VM35047:3 Zainab Nazar Khan
VM35047:3 Zahid Khatri
VM35047:3 Bhavi Dodhia
VM35047:3 Heenal Amin
VM35047:3 Hemant Tailor
VM35047:3 Manoj Kerai
VM35047:3 Izzy Langdon
VM35047:3 Nishma Varsani
VM35047:3 Sarah Husain
VM35047:3 Rosie Mac
VM35047:3 Stacy Lee
VM35047:3 Admas Haile
VM35047:3 Jamie Derek McCarthy
VM35047:3 Reena Patel
VM35047:3 Jaymini Naran-Mistry
VM35047:3 Sunit Patel
VM35047:3 Teena Varsani
VM35047:3 Saskia Dean
VM35047:3 Furat Ashraf
VM35047:3 Sapna S
VM35047:3 Loga Si
VM35047:3 Hem Raheja
VM35047:3 Ayesha Alam Malik
VM35047:3 Nigam Patel
VM35047:3 Shav Wijesuriya
VM35047:3 Huda Salmasi
VM35047:3 Khushboo Patel
VM35047:3 Caroline Kemp
VM35047:3 Dhanisha Jani
VM35047:3 Ellie Aldridge
VM35047:3 Charlotte Yan
VM35047:3 Harriet Waldegrave
VM35047:3 Reza Ghany
VM35047:3 Christine Smith
VM35047:3 Becci Burton
VM35047:3 Miran Shah
VM35047:3 Roshni Darshna Shah
VM35047:3 Anna Sella
VM35047:3 Sarah Nazarkhan
VM35047:3 Nurjuanis Zara
VM35047:3 Erika Bataillard