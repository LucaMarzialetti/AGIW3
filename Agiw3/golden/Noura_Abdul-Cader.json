{
    "PATTERN": {
        "tel": [],
        "addr": [],
        "email": []
    },
    "NER": {
        "LOC": [],
        "ORG": [],
        "PER": [
            "Alaa Abbas",
            "Noura Semary",
            "Mohiy Hadhoud",
            "Publisher"
        ]
    },
    "url": "http://core.ac.uk/display/22191073"
}